### PR TITLE
Fix error when location is null

### PR DESCRIPTION
**Pull request by miguelangel-nubla**

### CODE REVIEW

File: src/aftership-card.ts [hunk 1]
```diff
@@ -111,7 +111,7 @@ class AftershipCard extends LitElement {
                 </paper-item-body>
                 <paper-item-body class="last">
                   <div>
-                    ${item.last_checkpoint
+                    ${item.last_checkpoint.location
```
Comment (iantrich) [inline src/aftership-card.ts]: I think you should check both
```suggestion
                    ${item.last_checkpoint && item.last_checkpoint.location
```

Comment (miguelangel-nubla) [reply src/aftership-card.ts]: You are right, my mistake

**Comment by iantrich** [comment]
LGTM

**Comment by iantrich** [comment]
I never got around to setting up builds on this, so can you do a `npm run rollup` and commit the resulting `aftership-card.js`?

**Pull request closed without merging**